Gemfile

```diff
@@ -48,6 +48,7 @@ gem "honeycomb-beeline", "~> 2.7.0" # Monitoring and Observability gem
 gem "html_truncator", "~> 0.4" # Truncate an HTML string properly
 gem "htmlentities", "~> 4.3", ">= 4.3.4" # A module for encoding and decoding (X)HTML entities
 gem "httparty", "~> 0.19" # Makes http fun! Also, makes consuming restful web services dead easy
+gem "httpclient", "~> 2.8.3" # Gives something like the functionality of libwww-perl (LWP) in Ruby
```

Comment:
This was a dependency of Pusher before but when it got removed some thing started failing so I made the dependency explicit now.

Reply:
missed this in the main merge - added this back (probably without the comment, will add)